packages/reporter/cypress/e2e/suites.cy.ts

```diff
@@ -15,6 +15,7 @@ describe('suites', () => {
     cy.visit('/').then((win) => {
       win.render({
         runner,
+        studioEnabled: true,
```

Comment:
Same comment as above, this should only be enabled for a sub-set of tests

Reply:
https://github.com/cypress-io/cypress/pull/23544/commits/5ffeae71dc69efaa0ee7697d13f3a81ea57bc02c

There is a lot of duplication in the `beforeEach` for almost every single spec in `Reporter` - I think we should go through, refactor a lot (if not all) to component tests and introduce some shared utils to reduce duplication.

Also made a small improvement in ^ to use the correct prop for the runnerStore... looks like there isn't any type checking in reporter, we definitely need to get more proactive with type safety.